.jshintrc

```diff
@@ -46,7 +46,8 @@
     "moz"           : false,     // true: Allow Mozilla specific syntax (extends and overrides esnext features)
                                  // (ex: `for each`, multiple try/catch, function expression…)
     "evil"          : false,     // true: Tolerate use of `eval` and `new Function()`
-    "expr"          : false,     // true: Tolerate `ExpressionStatement` as Programs
+    // We allow the following due to "use strict" backwards compatibility bullshit
```

Comment:
Let's stay away from using words such as 'bullshit', they will not age well in the code base